tests/pipeline/conftest.py

```diff
@@ -7,14 +7,15 @@
 CONFIG_FILE = "tests/pipeline/fake_config.yaml"
 FAKE_AA_DATA_DIR = "fake_aa_dir"
 ISO3 = "abc"
+FAKE_IRI_AUTH = "def"
 
 
 @pytest.fixture(scope="session", autouse=True)
 def mock_aa_data_dir(session_mocker):
     """Mock out the AA_DATA_DIR environment variable."""
     session_mocker.patch.dict(
         "aatoolbox.config.pathconfig.os.environ",
-        {"AA_DATA_DIR": FAKE_AA_DATA_DIR},
+        {"AA_DATA_DIR": FAKE_AA_DATA_DIR, "IRI_AUTH": FAKE_IRI_AUTH},
```

Comment:
I know that it's annoying to make another fixture, but I feel like there should be a separate one for IRI only used in the IRI tests (that way you can control it separately if needed)